src/pytest/test_bond.py

```diff
@@ -68,6 +68,129 @@
     ),
 ]
 
+# bond.Quartic
+potential_tests += [
+    # test potential with sigma = epsilon = 0
+    PotentialTestCase(
+        hoomd.azplugins.bond.Quartic,
+        dict(
+            k=1434.3,
+            r_0=1.5,
+            b_1=-0.7589,
+            b_2=0,
+            U_0=67.2234,
+            sigma=0.0,
+            epsilon=0.0,
+            delta=0.0,
```

Comment:
Should we omit setting this parameter at least once if it has a default?
```suggestion
```

Reply:
Yes, I agree thank you for the feedback! I did have a few test cases where delta was omitted, but these were incompatible with the current test_energy_and_force(), failing at line 222, where it asserts that the parameters are unchanged.

```python
# test that parameters are still correct after attach runs
assert potential.params['A-A'] == potential_test.params
```

Thus, prior to making the PR, I removed this assertion and confirmed that the tests passed with delta omitted locally.
I then reinserted this assertion and removed the test cases for consistency.

Do you have a suggestion for how to approach this?

Reply:
Oh OK, I see! For now, leave it as is. Eventually, we will add automatic testing of the documentation, so having an example there that does not set delta explicitly will cover this case.